openff/recharge/utilities/toolkits.py

```diff
@@ -20,12 +21,14 @@ def _bond_key(index_a: int, index_b: int) -> Tuple[int, int]:
     return cast(Tuple[int, int], tuple(sorted((index_a, index_b))))
 
 
+@requires_oe_module("oechem")
 def _oe_match_smirks(
     smirks: str,
     molecule: "Molecule",
     is_atom_aromatic: Dict[int, bool],
     is_bond_aromatic: Dict[Tuple[int, int], bool],
     unique: bool,
+    kekulize: bool = False,
```

Comment:
If this is only supported by the RDKit function, should we error out if `kekulize=True` is passed?

Reply:
Or is it that kekulizing is just not necessary there?

Reply:
I think it's unnecessary and could result in spurious warnings -- 

```python
from openff.toolkit import Molecule
from openeye import oechem
benzene = Molecule.from_smiles("c1ccccc1")
rdmol = benzene.to_rdkit()
# AROMATIC
# AROMATIC
# AROMATIC
# AROMATIC
# AROMATIC
# AROMATIC
# SINGLE
# SINGLE
# SINGLE
# SINGLE
# SINGLE
# SINGLE
```
```python
oemol = benzene.to_openeye()
for bond in oemol.GetBonds():
    print(bond.GetOrder())
# 2
# 1
# 2
# 1
# 2
# 1
# 1
# 1
# 1
# 1
# 1
# 1
```

```python
oechem.OEKekulize(oemol)
# Warning: OEKekulize invalid integer bond type!
```

Reply:
This looks like a toolkit difference in conversion to rdkit/openeye -- the toolkit explicitly checks for aromaticity and sets BondType.AROMATIC for rdkit, but just sets the bond type from the (integer) bond order when converting to openeye. If the latter ever changes -- OpenEye does have an integer type (5) for aromaticity -- this function would need to change too.

Reply:
Actually the 5 type might not be used for just normal aromaticity. Either way, it looks like the bond order is already kekulized. These bonds don't match "-/=" when `.IsAromatic`, but do match if the aromaticity flag is set to False.

```python
mol = oechem.OEGraphMol()
oechem.OESmilesToMol(mol, "c1ccccc1")
for bond in mol.GetBonds():
    print(bond.GetOrder())
    print(bond.IsAromatic())
# 2
# True
# 1
# True
# 2
# True
# 1
# True
# 2
# True
# 1
# True
```